### PR TITLE
Heading of 'release notes' changed to 'Downloads'

### PR DESCRIPTION
**Pull request by NitishKumar06**
~Fix for~ relevant to https://github.com/kubernetes/website/issues/28054

Note: The lifecycle of this issue is frozen. Although one of the changes mentioned by @sftim [here](https://github.com/kubernetes/website/issues/28054#issuecomment-1169041782) has been applied in this PR.

Talking about other changes, It'd be great if I could get an answer to the question that I had:

Since _https://kubernetes.io/ should hyperlink to https://kubernetes.io/releases/download/_ . Do we need to update every link in any text format on K8s website?

### CODE REVIEW

File: content/en/docs/home/_index.md [hunk 1]
```diff
@@ -56,8 +56,8 @@ cards:
   description: Anyone can contribute, whether you're new to the project or you've been around a long time.
   button: Contribute to the docs
   button_path: /docs/contribute
-- name: release-notes
-  title: K8s Release Notes
+- name: Download
+  title: Downloads
   description: If you are installing Kubernetes or upgrading to the newest version, refer to the current release notes.
```
Comment (sftim) [inline content/en/docs/home/_index.md:61]: We should change this text too.

Comment (NitishKumar06) [reply content/en/docs/home/_index.md:61]: Sounds good! Thanks @sftim !

Comment (sftim) [reply content/en/docs/home/_index.md:61]: Once you have edited:
> If you are installing Kubernetes or upgrading to the newest version, refer to the current release notes.

then we can progress this PR.

Comment (NitishKumar06) [reply content/en/docs/home/_index.md]: @sftim It's done now!

File: content/en/docs/home/_index.md [hunk 2]
```diff
@@ -56,9 +56,9 @@ cards:
   description: Anyone can contribute, whether you're new to the project or you've been around a long time.
   button: Contribute to the docs
   button_path: /docs/contribute
-- name: release-notes
-  title: K8s Release Notes
-  description: If you are installing Kubernetes or upgrading to the newest version, refer to the current release notes.
+- name: Download
+  title: Downloads
```
Comment (shannonxtreme) [inline content/en/docs/home/_index.md:60]:
```suggestion
  title: Download Kubernetes
```

File: content/en/docs/home/_index.md [hunk 3]
```diff
@@ -56,9 +56,9 @@ cards:
...
+  description: If you are installing Kubernetes or upgrading to the newest version, refer to the Downloads section.
```
Comment (shannonxtreme) [inline content/en/docs/home/_index.md:61]:
```suggestion
  description: Install Kubernetes or upgrade to the newest version.
```
The button is the call to action. "Refer to the Downloads section" doesn't do anything unless we link that text to the downloads section, but that makes the button redundant. The Training section has similar language, keeping the button as the call to action

Comment (NitishKumar06) [reply content/en/docs/home/_index.md:61]: Thank you for suggestion!

**Comment by sftim** [comment]
I'm not sure why it's relevant that #28054 is frozen. Frozen means that we think the issue is important enough that we would never let it turn stale.

**Comment by NitishKumar06** [comment]
@sftim That issue has been opened a way long back ago! it's surprising that although some folks self-assigned themselves yet no one raised a PR in order to resolve that issue.

File: content/en/docs/home/_index.md (hunk 3 above)
Comment (NitishKumar06) [reply content/en/docs/home/_index.md]: @shannonxtreme the changes have been applied now!

**Comment by sftim** [comment]
Much appreciated @NitishKumar06

**Pull request merged**